### PR TITLE
=doc #15678 clarify log level limits events passed to actual logger

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: akka-docs/rst/java/logging.rst [hunk 1]
```diff
@@ -278,6 +278,10 @@ More fine grained log levels can be defined in the configuration of the SLF4J ba
 the ``logging-filter`` configuration property. It will filter the log events using the backend
 configuration (e.g. logback.xml) before they are published to the event bus.
 
+Note that if you set the `loglevel` to a higher level than "DEBUG", any DEBUG events will be filtered
+out already at the source and will never reach the logging backend, regardless of how it the backend
```
Comment (patriknw) [inline akka-docs/rst/java/logging.rst]: s/it the/the/

**Comment by patriknw** [comment]
LGTM

File: akka-docs/rst/java/logging.rst [hunk 2]
```diff
@@ -278,6 +278,10 @@ More fine grained log levels can be defined in the configuration of the SLF4J ba
 the ``logging-filter`` configuration property. It will filter the log events using the backend
 configuration (e.g. logback.xml) before they are published to the event bus.
 
+Note that if you set the `loglevel` to a higher level than "DEBUG", any DEBUG events will be filtered
```
Comment (drewhk) [inline akka-docs/rst/java/logging.rst]: .. warning::

**Comment by drewhk** [comment]
LGTM, but let's make it a proper warning in the markup

File: akka-docs/rst/java/logging.rst [hunk 3]
```diff
@@ -278,6 +278,11 @@ More fine grained log levels can be defined in the configuration of the SLF4J ba
 the ``logging-filter`` configuration property. It will filter the log events using the backend
 configuration (e.g. logback.xml) before they are published to the event bus.
 
+.. warning::
+  If you set the `loglevel` to a higher level than "DEBUG", any DEBUG events will be filtered
```
Comment (patriknw) [inline akka-docs/rst/java/logging.rst]: should be double back-ticks around `loglevel`

**Comment by patriknw** [comment]
please fix the minor format issue and squash

**Comment by patriknw** [comment]
Refs #15678

**Pull request merged**